src/Utility/StepHelper.php

```diff
@@ -65,22 +68,35 @@ protected function retryThrowable($callback, $timeout = 3)
     /**
      * Check if a step has a given tag
      *
-     * @param StepScope $event
+     * @param StepScope|ScenarioScope $event
      * @param string $tag
      * @return bool
      */
-    protected function stepHasTag(StepScope $event, $tag)
+    protected function stepHasTag($event, $tag)
```

Comment:
It's worth being careful of method signature changes for semver, but this is a protected method and also not a class which is particularly likely to have been extended

Reply:
Good catch. I didn't consider that.